From c741a131f2fb267b5b3bb333aca2453a07536535 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Fri, 20 Dec 2019 16:00:07 +0200
Subject: [PATCH] sdk_dpaa: ceetm: export the ceetm_tx symbol

In order to enable building the driver as a module, export the ceetm_tx
symbol for the DPAA Ethernet driver to use.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c
@@ -2077,6 +2077,7 @@ drop:
 	dev_kfree_skb_any(skb);
 	return NET_XMIT_SUCCESS;
 }
+EXPORT_SYMBOL(ceetm_tx);
 
 static int __init ceetm_register(void)
 {
